appveyor.yml

```diff
@@ -29,9 +29,8 @@ install:
   - ruby --version
   - gem --version
   - bundler --version
-  - gem uninstall bundler -a -x
   - gem update --system %OMNIBUS_RUBYGEMS% || gem update --system %OMNIBUS_RUBYGEMS% || gem update --system %OMNIBUS_RUBYGEMS%
-  - gem install bundler -v %OMNIBUS_BUNDLER% --quiet --no-ri --no-rdoc || gem install bundler -v %OMNIBUS_BUNDLER% --quiet --no-ri --no-rdoc || gem install bundler -v %OMNIBUS_BUNDLER% --quiet --no-ri --no-rdoc
+  - gem install bundler -v %OMNIBUS_BUNDLER% --quiet --no-ri --no-rdoc --force || gem install bundler -v %OMNIBUS_BUNDLER% --quiet --no-ri --no-rdoc --force || gem install bundler -v %OMNIBUS_BUNDLER% --quiet --no-ri --no-rdoc --force
```

Comment:
I don't think we need to do this any more. Just leave the system bundler in place - appveyor is building it in now.

Reply:
We're still installing a specific version of bundler on Travis. I figured the point here is to always test with the version of bundler in `omnibus_overrides.rb`. This line does successfully give us 1.15.4 rather than the 1.16.0 that Appveyor is shipping.

Reply:
If we don't care what version of bundler is in use, then yah sure.

Reply:
i think we should always ship the latest (and we should do the same in omnibus, eventually). Ruby 2.5 will take this out of our hands anyway.

Reply:
🚅